Source/WebKit/UIProcess/ios/forms/WKDateTimeInputControl.mm

```diff
@@ -87,6 +87,8 @@ - (id)initWithView:(WKContentView *)view inputType:(WebKit::InputType)inputType
         return nil;
 
     RetainPtr maximumDateFormatter = adoptNS([[NSDateFormatter alloc] init]);
+    [maximumDateFormatter setLocale:[NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"]];
+    [maximumDateFormatter setTimeZone:[NSTimeZone timeZoneForSecondsFromGMT:0]];
     [maximumDateFormatter setDateFormat:kDateTimeFormatString];
     RetainPtr maximumDate = [maximumDateFormatter dateFromString:@"10000-12-31T23:59"]; // UIDatePicker cannot have more than 10,000 selectable years
     _dateInterval = adoptNS([[NSDateInterval alloc] initWithStartDate:[NSDate distantPast] endDate:maximumDate.get()]);
```

Comment:
should there be a RELEASE_ASSERT added here to make sure maximumDate is never nil?

Reply:
Also the existing crash log is very clear: "endDate is nil".